### PR TITLE
upgrade cse import runner and cse export runner to farao-dependencies…

### PR DESCRIPTION
**Pull request by kahyami**
… v1.33.0

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by danielthirion — DISMISSED** [review]

File: cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java [hunk 1]
```diff
@@ -42,17 +45,29 @@ Network importNetwork(String cgmUrl) {
...
         InputStream cracInputStream = openUrlStream(cracUrl);
-        CseCracImporter cseCracImporter = new CseCracImporter();
-        return cseCracImporter.importNativeCrac(cracInputStream);
+        return importNativeCrac(cracInputStream);
+    }
+
+    CRACDocumentType importNativeCrac(InputStream inputStream) {
+        CRACDocumentType cracDocumentType;
+        try {
+            cracDocumentType = JAXBContext.newInstance(CRACDocumentType.class)
```
Comment (vbochetRTE) [inline cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java:56]: You could have put the return directly inside of the try and remove the useless variable declaration and assignment.

File: cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java [hunk 2]
```diff
@@ -42,17 +45,29 @@ Network importNetwork(String cgmUrl) {
...
 
-    CseCrac importCseCrac(String cracUrl) {
+    CRACDocumentType importCseCrac(String cracUrl) {
         InputStream cracInputStream = openUrlStream(cracUrl);
-        CseCracImporter cseCracImporter = new CseCracImporter();
-        return cseCracImporter.importNativeCrac(cracInputStream);
+        return importNativeCrac(cracInputStream);
+    }
+
+    CRACDocumentType importNativeCrac(InputStream inputStream) {
```
Comment (vbochetRTE) [inline cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java:53]: This method looks like it should be extracted to a common utility class: such a treatment is also declared and used in the `FileImporter` from the import_runner and in the `BusBarChangePreProcessor`.

File: cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/FileImporter.java [hunk 3]
```diff
@@ -63,23 +64,36 @@ public Network importNetwork(String cgmUrl) {
         return Network.read(FileUtil.getFilenameFromUrl(cgmUrl), openUrlStream(cgmUrl));
     }
 
-    public CseCrac importCseCrac(String cracUrl) {
+    public CRACDocumentType importCseCrac(String cracUrl) {
         InputStream cracInputStream = openUrlStream(cracUrl);
-        CseCracImporter cseCracImporter = new CseCracImporter();
-        return cseCracImporter.importNativeCrac(cracInputStream);
+
+        CRACDocumentType cracDocumentType;
```
Comment (vbochetRTE) [inline cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/FileImporter.java:70]: Duplicate from export_runner's `FileImporter`.

File: cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/busbar_change/BusBarChangePreProcessor.java [hunk 4]
```diff
@@ -42,21 +43,44 @@ private BusBarChangePreProcessor() {
         // utility class
     }
 
+    private static CRACDocumentType importNativeCrac(InputStream inputStream) {
```
Comment (vbochetRTE) [inline cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/busbar_change/BusBarChangePreProcessor.java:46]: Duplicate from export_runner's `FileImporter`.

File: cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java (hunk 2 above)
Comment (vbochetRTE) [reply cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java:53]: As discussed, this point should be handled in a separate ticket dedicated to refactoring. Won't be changed in this PR.

File: cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/FileImporter.java (hunk 3 above)
Comment (vbochetRTE) [reply cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/FileImporter.java:70]: As discussed, this point should be handled in a separate ticket dedicated to refactoring. Won't be changed in this PR.

File: cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/busbar_change/BusBarChangePreProcessor.java (hunk 4 above)
Comment (vbochetRTE) [reply cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/busbar_change/BusBarChangePreProcessor.java:46]: As discussed, this point should be handled in a separate ticket dedicated to refactoring. Won't be changed in this PR.

File: cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/util/FlowEvaluator.java [hunk 5]
```diff
@@ -35,6 +35,6 @@ public static FlowResult evaluate(Crac crac, Network network, Unit unit) {
             .withSensitivityProviderName(SensitivityAnalysis.find().getName())
             .build();
         SystematicSensitivityResult sensitivityResult = systematicSensitivityInterface.run(network);
-        return new FlowResultImpl(sensitivityResult, null, null);
+        return new FlowResultImpl(sensitivityResult, (FlowResult) null, null);
```
[the remark this answers is not in the document]
Comment (kahyami) [reply cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/util/FlowEvaluator.java:38]: cast is is mandatory here 
cast

**Review by vbochetRTE — APPROVED** [review]

**Review by danielthirion — APPROVED** [review]

**Pull request merged**